PurchasesCoreSwift/Logging/Strings/IdentityStrings.swift

```diff
@@ -21,9 +22,11 @@ import Foundation
     "already cached. No action will be taken."}
     @objc public var creating_alias_success: String { "Alias created" }
     @objc public var login_success: String { "Log in successful" }
-    @objc public var logging_out_user: String { "Logging out user %@" }
+    @objc public var log_out_called_for_user: String { "Log out called for user %@" }
```

Comment:
👍 i've noticed from users' debug logs that we have more cases like this. i'll keep an eye out